### DISCUSSION TITLE
Capture clip on motion detection

### DISCUSSION THREAD

**Discussion by ramiws** [jameszah/ESP32-CAM-Video-Telegram#45] [Ideas]
Hi,

This is so cool and avoid the need for server side for remote access. But it would be great to have it trigger clip recording when motion is detected in the frame.  I'm happy to buy anyone who would pick this a coffee and ice cream

**Comment by ramiws** [comment]
The pir doesn't work in my case because the camera is monitoring a construction site and the sun gives a lot of ir light

**Comment by FBMinis** [comment]
You can replace the PIR with a radar sensor - those rectangular green boards, i forgot their name. BigClive, Drone Bot Workshop and Andreas Speiss have review videos on youtube.

**Comment by FBMinis** [comment]
I've tried this and it work ok:

https://github.com/eloquentarduino/EloquentArduino
https://eloquentarduino.com/esp32-cam-motion-detection/
